recipes/gunicorn/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "gunicorn" %}
+{% set version = "19.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/benoitc/gunicorn/archive/{{ version }}.tar.gz
+  sha256: 1e0de4957bea60bfcff5215664bdfc3cf02c78e2aae9586766a9b4b437aebbb0
+  patches:
+    # See https://github.com/conda-forge/staged-recipes/pull/885#discussion_r68602712
+    # git diff --no-prefix -- gunicorn/workers/_gaiohttp.py /dev/null
+    - no_gaiohttp.patch  # [py2k]
```

Comment:
Thanks for the clarification, @RSully. SGTM. Could we please update the link in `meta.yaml` with this one as it works better?